tests/filecheck/parser-printer/duplicate_attribute_keys.mlir

```diff
@@ -0,0 +1,9 @@
+// RUN: xdsl-opt %s --parsing-diagnostics --split-input-file | filecheck %s
+
+"test.op"() {key1, key1} : () -> ()
```

Comment:
I believe we aim for putting the `CHECK` directive on the line above the actual code now. (This isn't followed everywhere but may as well try to have a convention)